build/test/OverlaySdkOnLKG.targets

```diff
@@ -2,22 +2,13 @@
   <Target Name="OverlaySdkOnLKG">
     <ItemGroup>
       <OverlaySDK Include="$(DotNetCoreSdkLKGPublishDirectory)/**/*" />
-      <OverlayCLI Include="$(OutputDirectory)/**/*" />
+      <OverlayCLI Include="$(CliOutputDirectory)/**/*" />
```

Comment:
I'm having trouble understanding why the whole overlay process is changed by this. Can you describe what was happening before vs. what happens after this change?

Reply:
@nguerrera 
The overlay process was using the CLI version instead of the SDK-LKG version - the SDK versioned folder had a CLI version.